src/utils/urlPaths.ts

```diff
@@ -0,0 +1,35 @@
+export enum UrlPathTemplate {
```

Comment:
enig, men synes det er rart å postfix'e med `Template`. Det er jo ikke en template, det er standarden vår. Kan vi heller kalle det `UrlPath`?

Reply:
Hvis vi kaller den `UrlPath` virker det som at alle kan brukes direkte også, men feks `RegistrationLandingPage = '/registration/:identifier/public'` er ikke en skikkelig path før vi erstatter `:identifier` med en faktisk id. tror dette blir litt rart uansett hvordan vi gjør det, så kan godt fjerne `Template` for min del

Reply:
åja, sånn ja. ok, men da er det greit!

Reply:
så vi må bare passe på selv at vi ikke roter til noe der. Vurderte å bare eksportere noe ala
```
export const urlPath = {
    landingPage: {
        template: UrlPathTemplate.RegistrationLandingPage,
        actual: (identifier: string) => UrlPathTemplate.RegistrationLandingPage.replace(':identifier',  encodeURIComponent(identifier))
    }
   (...)
}
```
men syns ikke det ble så mye bedre heller, selv om det kanskje er litt mer forklarende

Reply:
nei, det ble mer rotete synes jeg